environment-pipelines/tests/unit.tftest.hcl

```diff
@@ -126,6 +126,13 @@ override_data {
   }
 }
 
+override_data {
+  target = data.aws_iam_policy_document.ecs
+  values = {
+    json = "{\"Sid\": \"ECS\"}"
+  }
+}
+
```

Comment:
This does not need to be used in any assertions.

I know this area is a bit weak in `terrarfom test` , but I think we can at least assert the the correct policy document was used...

https://github.com/uktrade/terraform-platform-modules/blob/d8ba1af68c4314352ea440f0500edd4cfb69ef7a/opensearch/tests/unit.tftest.hcl#L446-L450

Reply:
Tests added.